conf/routes

```diff
@@ -42,6 +42,7 @@ GET         /collection/*collectionId                controllers.FaciaToolContro
 POST        /edits                                   controllers.FaciaToolController.collectionEdits
 GET         /config                                  controllers.FaciaToolController.getConfig
 GET         /defaults                                controllers.DefaultsController.configuration()
+GET         /tags                                    controllers.FaciaToolController.getTags()
```

Comment:
`GET    /metadata/collection ` because they are not tags and in case we ever need other types of metadata